packages/bridge-ui-v2/src/libs/bridge/types.ts

```diff
@@ -9,6 +9,7 @@ export enum MessageStatus {
   RETRIABLE,
   DONE,
   FAILED,
+  NewOnlyOwner,
```

Comment:
Where does this status come from? MessageStatus in the contract is unchanged and this should mirror it.

Reply:
Synced with Jeff. NewOnlyOwner is used internally in relayer. Removed.